resources/migrations/20171226182259-hide-user.down.sql

```diff
@@ -0,0 +1,35 @@
+BEGIN;
+
+ALTER TABLE users
+    DROP COLUMN is_hidden;
+
+-- restore the previous version of the view
```

Comment:
here and below, I had to redefine existing DB view to add non-hidden user clause.